docs/4. Product Features/03. Flows/2. Flow Actions/04. Wait for the contact to respond.md

```
@@ -1,236 +1,266 @@
 
-![image](https://user-images.githubusercontent.com/32592458/218255777-a8b9b9be-0c3c-4467-850a-e7d7a764ecfc.png)
+---
 
+## Matches Regex
 
+This option is used to validate a response against a specific format using a regular expression (regex).
+It is especially useful when inputs need to follow strict patterns, such as dates, ID numbers, or custom codes.
 
-It will be shown in the chat window like the below example.
+Example: To ensure a date of birth is entered in the format DD/MM/YYYY, use the following regex:
+^(0[1-9]|[12][0-9]|3[01])/(0[1-9]|1[0-2])/[0-9]{4}$
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "0-9"

(MD052, reference-links-images)